relay-general/src/protocol/types.rs

```diff
@@ -976,9 +976,9 @@ impl FromValue for Timestamp {
     fn from_value(value: Annotated<Value>) -> Annotated<Self> {
         let rv = match value {
             Annotated(Some(Value::String(value)), mut meta) => {
-                let parsed = match value.parse::<NaiveDateTime>() {
+                let parsed = match NaiveDateTime::parse_from_str(&value, "%Y-%m-%dT%H:%M:%S%.f") {
```

Comment:
```suggestion
                // Use the same format as `NaiveDateTime::FromStr`: https://docs.rs/chrono/0.4.19/chrono/naive/struct.NaiveDateTime.html#impl-FromStr
                let parsed = match NaiveDateTime::parse_from_str(&value, "%Y-%m-%dT%H:%M:%S%.f") {
```

Reply:
This is lenient against spaces, but it's not obvious from the documentation or the format string. Can you please add a test case for something like `"2015-  09-18T23:56:04"`?

Reply:
@jan-auer is this still relevant? are you saying FromStr is lenient as well?